daily_milestone.go

```diff
@@ -1,29 +1,65 @@
 package main
 
-import "flag"
+import (
+	"flag"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var logger *log.Logger
+
+//LoggerSetup Initialization
+func LoggerSetup(info io.Writer) {
+	logger = log.New(info, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
+}
+
+func getProjectId(BaseUrl string, Token string, Projectname string, Namespace string) string {
+	headers := map[string]string{"PrivateToken": Token}
+	url := "https://" + BaseUrl + "/projects"
 
-func get_project_id(base_url string, token string, projectname string, namespace string) string {
-	headers := map[string]string{"PrivateToken": token}
-	url := "https://" + base_url + "/projects"
 	page := 1
+	strPage := strconv.Itoa(page)
+	s := []string{url, "?page=", strPage}
+	completeUrl := strings.Join(s, "")
```

Comment:
You might even be able to shorten completeUrl to just url. But that is more subjective. Some people prefer shorter variable names within the functions as each function usually is quite selfcontained and therefore variables are selfexplained mostly.